`build.cmd Clr+Libs` with Insider Preview Windows SDK fails during builds of libunwind

I'm currently setting my laptop up for building runtime. When I attempt to build the runtime with `build.cmd Clr+Libs`, the build fails with some weird compilation error. It appears that the CoreCLR Component, for some reason, is being built for Linux on my Windows machine - I'm not exactly sure why this is happening. Helps would be appreciated 🙂 

I'm on Windows 10 Home Insiders Preview (Build 20175) for x64, ko-KR. 

<details><summary>Build output:</summary>

```
❯ .\build.cmd Clr+Libs
  복원할 프로젝트를 확인하는 중...
  'coverlet.console' 도구(버전 '1.7.2')가 복원되었습니다. 사용 가능한 명령: coverlet
  'dotnet-reportgenerator-globaltool' 도구(버전 '4.5.8')가 복원되었습니다. 사용 가능한 명령: reportgenerator
  'microsoft.dotnet.xharness.cli' 도구(버전 '1.0.0-prerelease.20352.2')가 복원되었습니다. 사용 가능한 명령: xharness

  복원했습니다.
  복원할 모든 프로젝트가 최신 상태입니다.
  복원할 프로젝트를 확인하는 중...
  복원할 모든 프로젝트가 최신 상태입니다.
  System.Private.CoreLib -> C:\Users\gotos\source\repos\runtime\artifacts\bin\coreclr\Windows_NT.x64.Debug\IL\System.Private.CoreLib.dll
  BUILD: Starting Build at 15:01:27.29
  BUILD: Searching for Visual Studio installation
  BUILD: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\Tools\VsDevCmd.bat"
  **********************************************************************
  ** Visual Studio 2019 Developer Command Prompt v16.7.0-pre.5.0
  ** Copyright (c) 2020 Microsoft Corporation
  **********************************************************************
  BUILD: Commencing CoreCLR product build
  BUILD: Checking prerequisites
  BUILD: Using CMake from C:\Program Files\CMake\bin\cmake.exe
  BUILD: Number of processor cores 4
  BUILD: Generating native version headers
  C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\MSBuild.dll /nologo -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll -maxcpucount /m -verbosity:m /v:minimal /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\GenerateVersionHeaders_Windows_NT__x64__Debug.binlog /clp:Summary /clp:nosummary /nr:True /p:ContinuousIntegrationBuild=False /p:TreatWarningsAsErrors=true /p:NativeVersionFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\_version.h /p:RuntimeVersionFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\runtime_version.h /p:TargetOS=Windows_NT /p:Configuration=Debug /p:TargetArchitecture=x64 /restore /t:GenerateRuntimeVersionFile /warnaserror C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\eng\empty.csproj
    복원할 프로젝트를 확인하는 중...
    복원할 모든 프로젝트가 최신 상태입니다.
  BUILD: Restoring the OptimizationData Package
  C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\MSBuild.dll /nologo -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll -maxcpucount /m -verbosity:m /v:minimal /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\OptRestore_Windows_NT__x64__Debug.binlog /clp:Summary /clp:nosummary /nr:True /nodereuse:false /p:ContinuousIntegrationBuild=False /p:TreatWarningsAsErrors=true /p:TargetOS=Windows_NT /p:Configuration=Debug /p:TargetArchitecture=x64 /t:Restore /warnaserror C:\Users\gotos\source\repos\runtime\src\coreclr\src\.nuget\optdata\optdata.csproj
    복원할 프로젝트를 확인하는 중...
    복원할 모든 프로젝트가 최신 상태입니다.
  C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\MSBuild.dll /nologo -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll -maxcpucount /m -verbosity:m /v:minimal /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\PgoVersionRead_Windows_NT__x64__Debug.binlog /clp:Summary /clp:nosummary /nr:True /p:ContinuousIntegrationBuild=False /p:TreatWarningsAsErrors=true /p:TargetOS=Windows_NT /p:Configuration=Debug /p:TargetArchitecture=x64 /p:PgoDataPackagePathOutputFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\Windows_NT.x64.Debug\optdatapath.txt /t:DumpPgoDataPackagePath /warnaserror C:\Users\gotos\source\repos\runtime\src\coreclr\src\.nuget\optdata\optdata.csproj
    optimizationPGOCoreCLR Package path: C:\Users\gotos\.nuget\packages\optimization.windows_nt-x64.pgo.coreclr\99.99.99-master-20200228.3 written to: C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\Windows_NT.x64.Debug\optdatapath.txt
  BUILD: Commencing build of native components for Windows_NT.x64.Debug
  BUILD: Using environment: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\Tools\\..\..\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
  **********************************************************************
  ** Visual Studio 2019 Developer Command Prompt v16.7.0-pre.5.0
  ** Copyright (c) 2020 Microsoft Corporation
  **********************************************************************
  [vcvarsall.bat] Environment initialized for: 'x86_x64'
  BUILD: Regenerating the Visual Studio solution
  -- Selecting Windows SDK version 10.0.20175.0 to target Windows 10.0.
  -- VS_PLATFORM_TOOLSET is v142
  -- VS_PLATFORM_NAME is x64
  Read file version from native version header at 'C:/Users/gotos/source/repos/runtime/src/coreclr/../../artifacts/obj/coreclr/_version.h'.
  -- Configuring done
  -- Generating done
  -- Build files have been written to: C:/Users/gotos/source/repos/runtime/artifacts/obj/coreclr/Windows_NT.x64.Debug
  C:/Program Files (x86)/Microsoft Visual Studio/2019/Preview/MSBuild/Current/Bin/MSBuild.exe /nologo /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\CoreCLR_Windows_NT__x64__Debug.binlog /flp1:WarningsOnly;LogFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\CoreCLR_Windows_NT__x64__Debug.wrn /flp2:ErrorsOnly;LogFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\CoreCLR_Windows_NT__x64__Debug.err /flp:Verbosity=normal;LogFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\CoreCLR_Windows_NT__x64__Debug.log /m /p:Configuration=Debug /p:Platform=x64 /p:VisualStudioVersion=16.0 /v:m install.vcxproj
    mdruntimerw_wks.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\enc\mdruntimerw_wks.dir\Debug\mdruntimerw_wks.lib
    mdruntime_wks.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\runtime\mdruntime_wks.dir\Debug\mdruntime_wks.lib
    mdhotdata_full.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\hotdata\mdhotdata_full.dir\Debug\mdhotdata_full.lib
    InjectResource.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\tools\InjectResource\Debug\InjectResource.exe
    GenClrDebugResource.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\tools\GenClrDebugResource\Debug\GenClrDebugResource.exe
    mdcompiler_wks.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\compiler\mdcompiler_wks.dir\Debug\mdcompiler_wks.lib
    interop.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\interop\interop.dir\Debug\interop.lib
    gcinfo.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\gcinfo\gcinfo.dir\Debug\gcinfo.lib
    mscordacobj.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscordac\mscordacobj.dir\Debug\mscordacobj.lib
    clrjit_static.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\jit\static\clrjit_static.dir\Debug\clrjit_static.lib
    corguids.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\inc\corguids.dir\Debug\corguids.lib
    ceefgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\ceefilegen\ceefgen.dir\Debug\ceefgen.lib
    debug-pal.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\debug-pal\debug-pal.dir\Debug\debug-pal.lib
    ildbsymlib.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\ildbsymlib\ildbsymlib.dir\Debug\ildbsymlib.lib
    System.Globalization.Native-Static.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\libraries-native\System.Globalization.Native\System.Globalization.Native-Static.dir\Debug\System.Globalization.Native-Static.lib
    bcltype.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\classlibnative\bcltype\bcltype.dir\Debug\bcltype.lib
    dactablegen -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ToolBox\SOS\DacTableGen\Debug\dactablegen.exe
    v3binder.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\binder\v3binder.dir\Debug\v3binder.lib
    cordbee_wks.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\ee\wks\cordbee_wks.dir\Debug\cordbee_wks.lib
    utilcode.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\utilcode\utilcode.dir\Debug\utilcode.lib
    eventpipe.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\eventing\eventpipe\eventpipe.dir\Debug\eventpipe.lib
    cee_wks_core.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\wks\cee_wks_core.dir\Debug\cee_wks_core.lib
    comfloat_wks.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\classlibnative\float\comfloat_wks.dir\Debug\comfloat_wks.lib
    daccess.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\daccess\Debug\daccess.lib
    cordbdi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\di\Debug\cordbdi.lib
    mdcompiler-dbi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\compiler\Debug\mdcompiler-dbi.lib
    mdhotdata-staticcrt.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\hotdata\Debug\mdhotdata-staticcrt.lib
    mdruntime-dbi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\runtime\Debug\mdruntime-dbi.lib
    mddatasource_dbi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\datasource\Debug\mddatasource_dbi.lib
    mdruntimerw-dbi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\enc\Debug\mdruntimerw-dbi.lib
    utilcodestaticnohost.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\utilcode\Debug\utilcodestaticnohost.lib
    cee_wks.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\wks\cee_wks.dir\Debug\cee_wks.lib
    cee_wks_mergeable.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\wks\cee_wks_mergeable.dir\Debug\cee_wks_mergeable.lib
    mscordbi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscordbi\Debug\mscordbi.dll
    dbgutil.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\dbgutil\Debug\dbgutil.lib
    coreclr.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscoree\coreclr\Debug\coreclr.dll
    unwinder_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\unwinder\Debug\unwinder_dac.lib
    mdhotdata_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\hotdata\Debug\mdhotdata_dac.lib
    cordbee_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\ee\dac\Debug\cordbee_dac.lib
    mdruntime_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\runtime\Debug\mdruntime_dac.lib
    mdcompiler_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\compiler\Debug\mdcompiler_dac.lib
    utilcode_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\utilcode\Debug\utilcode_dac.lib
    cee_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\Debug\cee_dac.lib
    mdruntimerw_dac.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\enc\Debug\mdruntimerw_dac.lib
    mscordaccore.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscordac\Debug\mscordaccore.dll
    corzap_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\zap\Debug\corzap_crossgen.lib
    CoreConsole.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\hosts\coreconsole\Debug\CoreConsole.exe
    clretwrc.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\clretwrc\Debug\clretwrc.dll
    CoreShim.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\hosts\coreshim\Debug\CoreShim.dll
    createdump.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\createdump\Debug\createdump.exe
    CoreRun.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\hosts\corerun\Debug\CoreRun.exe
    clrgc.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\gc\Debug\clrgc.dll
    cee_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\crossgen\Debug\cee_crossgen.lib
    gcinfo_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\gcinfo\Debug\gcinfo_crossgen.lib
    mscorlib_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\vm\crossgen\Debug\mscorlib_crossgen.lib
    v3binder_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\binder\Debug\v3binder_crossgen.lib
    mdhotdata_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\hotdata\Debug\mdhotdata_crossgen.lib
    mdruntime_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\runtime\Debug\mdruntime_crossgen.lib
    clrjit.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\jit\standalone\Debug\clrjit.dll
    mdruntimerw_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\enc\Debug\mdruntimerw_crossgen.lib
    mdcompiler_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\compiler\Debug\mdcompiler_crossgen.lib
    gcinfo_arm64.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\gcinfo\gcinfo_arm64\Debug\gcinfo_arm64.lib
    utilcode_crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\utilcode\Debug\utilcode_crossgen.lib
    debugshim.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\shim\Debug\debugshim.lib
    mdcompiler_ppdb.vcxproj -> C:\Users\gotos\I'm trying to set up source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\compiler\Debug\mdcompiler_ppdb.lib
    gcsample.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\gc\sample\Debug\gcsample.exe
    dbgshim.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\dbgshim\Debug\dbgshim.dll
    jitinterface.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\tools\aot\jitinterface\Debug\jitinterface.dll
    mdhotdata_ppdb.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\hotdata\Debug\mdhotdata_ppdb.lib
    linuxnonjit.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\jit\linuxnonjit\Debug\linuxnonjit.dll
    mdruntimerw_ppdb.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\enc\Debug\mdruntimerw_ppdb.lib
    mdruntime_ppdb.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\runtime\Debug\mdruntime_ppdb.lib
    mdstaticapi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\staticmd\Debug\mdstaticapi.lib
    mdstaticapi_ppdb.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\md\staticmd\Debug\mdstaticapi_ppdb.lib
    mscorrc.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscorrc\Debug\mscorrc.dll
    mscorpe.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscorpe\Debug\mscorpe.lib
    protononjit.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\jit\protononjit\Debug\protononjit.dll
    ildasm.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ildasm\exe\Debug\ildasm.exe
    mcs.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ToolBox\superpmi\mcs\Debug\mcs.exe
    superpmi.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ToolBox\superpmi\superpmi\Debug\superpmi.exe
    superpmi-shim-collector.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ToolBox\superpmi\superpmi-shim-collector\Debug\superpmi-shim-collector.dll
    superpmi-shim-simple.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ToolBox\superpmi\superpmi-shim-simple\Debug\superpmi-shim-simple.dll
    superpmi-shim-counter.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ToolBox\superpmi\superpmi-shim-counter\Debug\superpmi-shim-counter.dll
    coreclr_static.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\dlls\mscoree\coreclr\Debug\coreclr_static.lib
    crossgen.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\tools\crossgen\Debug\crossgen.exe
    ilasm.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\ilasm\Debug\ilasm.exe
    runtimeinfo.vcxproj -> C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Windows_NT.x64.Debug\src\debug\runtimeinfo\Debug\runtimeinfo.lib
    -- Install configuration: "Debug"
  0개 파일이 복사되었습니다.
  BUILD: Build succeeded.  Finished at 15:02:09.27
  BUILD: Product binaries are available at C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\bin\coreclr\Windows_NT.x64.Debug
  BUILD: Starting Build at 15:02:09.32
  BUILD: Searching for Visual Studio installation
  BUILD: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\Tools\VsDevCmd.bat"
  **********************************************************************
  ** Visual Studio 2019 Developer Command Prompt v16.7.0-pre.5.0
  ** Copyright (c) 2020 Microsoft Corporation
  **********************************************************************
  BUILD: Commencing CoreCLR product build
  BUILD: Checking prerequisites
  BUILD: Using CMake from C:\Program Files\CMake\bin\cmake.exe
  BUILD: Number of processor cores 4
  BUILD: Generating native version headers
  C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\MSBuild.dll /nologo -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll -maxcpucount /m -verbosity:m /v:minimal /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\GenerateVersionHeaders_Linux__x64__Debug.binlog /clp:Summary /clp:nosummary /nr:True /p:ContinuousIntegrationBuild=False /p:TreatWarningsAsErrors=true /p:NativeVersionFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\_version.h /p:RuntimeVersionFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\runtime_version.h /p:TargetOS=Linux /p:Configuration=Debug /p:TargetArchitecture=x64 /restore /t:GenerateRuntimeVersionFile /warnaserror C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\eng\empty.csproj
    복원할 프로젝트를 확인하는 중...
    복원할 모든 프로젝트가 최신 상태입니다.
  BUILD: Restoring the OptimizationData Package
  C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\MSBuild.dll /nologo -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll -maxcpucount /m -verbosity:m /v:minimal /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\OptRestore_Linux__x64__Debug.binlog /clp:Summary /clp:nosummary /nr:True /nodereuse:false /p:ContinuousIntegrationBuild=False /p:TreatWarningsAsErrors=true /p:TargetOS=Linux /p:Configuration=Debug /p:TargetArchitecture=x64 /t:Restore /warnaserror C:\Users\gotos\source\repos\runtime\src\coreclr\src\.nuget\optdata\optdata.csproj
    복원할 프로젝트를 확인하는 중...
    복원할 모든 프로젝트가 최신 상태입니다.
  C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\MSBuild.dll /nologo -distributedlogger:Microsoft.DotNet.Tools.MSBuild.MSBuildLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll*Microsoft.DotNet.Tools.MSBuild.MSBuildForwardingLogger,C:\Users\gotos\source\repos\runtime\.dotnet\sdk\5.0.100-preview.8.20362.3\dotnet.dll -maxcpucount /m -verbosity:m /v:minimal /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\PgoVersionRead_Linux__x64__Debug.binlog /clp:Summary /clp:nosummary /nr:True /p:ContinuousIntegrationBuild=False /p:TreatWarningsAsErrors=true /p:TargetOS=Linux /p:Configuration=Debug /p:TargetArchitecture=x64 /p:PgoDataPackagePathOutputFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\Linux.x64.Debug\optdatapath.txt /t:DumpPgoDataPackagePath /warnaserror C:\Users\gotos\source\repos\runtime\src\coreclr\src\.nuget\optdata\optdata.csproj
    optimizationPGOCoreCLR Package path: C:\Users\gotos\.nuget\packages\optimization.linux-x64.pgo.coreclr\99.99.99-master-20200228.3 written to: C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\obj\coreclr\Linux.x64.Debug\optdatapath.txt
  BUILD: Commencing build of cross architecture native components for Linux.x64.Debug
  BUILD: Using environment: "C:\Program Files (x86)\Microsoft Visual Studio\2019\Preview\Common7\Tools\\..\..\VC\Auxiliary\Build\vcvarsall.bat" x86_amd64
  **********************************************************************
  ** Visual Studio 2019 Developer Command Prompt v16.7.0-pre.5.0
  ** Copyright (c) 2020 Microsoft Corporation
  **********************************************************************
  [vcvarsall.bat] Environment initialized for: 'x86_x64'
  -- Selecting Windows SDK version 10.0.20175.0 to target Windows 10.0.
  -- VS_PLATFORM_TOOLSET is v142
  -- VS_PLATFORM_NAME is x64
  PGO data file NOT found: C:\Users\gotos\.nuget\packages\optimization.linux-x64.pgo.coreclr\99.99.99-master-20200228.3\data\clrjit.pgd
  Read file version from native version header at 'C:/Users/gotos/source/repos/runtime/src/coreclr/../../artifacts/obj/coreclr/_version.h'.
  PGO data file NOT found: C:\Users\gotos\.nuget\packages\optimization.linux-x64.pgo.coreclr\99.99.99-master-20200228.3\data\coreclr.pgd
  -- Configuring done
  -- Generating done
  -- Build files have been written to: C:/Users/gotos/source/repos/runtime/artifacts/obj/coreclr/Linux.x64.Debug/crossgen
  C:/Program Files (x86)/Microsoft Visual Studio/2019/Preview/MSBuild/Current/Bin/MSBuild.exe /nologo /bl:C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.binlog /flp1:WarningsOnly;LogFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.wrn /flp2:ErrorsOnly;LogFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.err /flp:Verbosity=normal;LogFile=C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.log /m /p:Configuration=Debug /p:Platform=x64 /p:VisualStudioVersion=16.0 /v:m install.vcxproj
    mempool.c
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(44,16): error C2143: syntax error: missing ')' before '(' [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(44,16): error C2143: syntax error: missing ')' before 'sizeof' [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(44,16): error C2091: function returns function [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(44,16): error C2143: syntax error: missing '{' before 'sizeof' [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(44,16): error C2059: syntax error: 'sizeof' [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(56,3): error C2065: 'sos_memory': undeclared identifier [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(56,3): error C2109: subscript requires array or pointer type [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(60,3): error C2065: 'sos_memory': undeclared identifier [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(60,3): error C2109: subscript requires array or pointer type [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(63,21): error C2065: 'sos_memory': undeclared identifier [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(63,25): error C2109: subscript requires array or pointer type [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  C:\Users\gotos\source\repos\runtime\src\coreclr\src\pal\src\libunwind\src\mi\mempool.c(63,3): warning C4033: '_UIx86_64__sos_alloc' must return a value [C:\Users\gotos\source\repos\runtime\artifacts\obj\coreclr\Linux.x64.Debug\crossgen\src\pal\src\libunwind\src\libunwind_xdac.vcxproj]
  BUILD: Error: cross-arch components build failed. Refer to the build log files for details.
      C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.log
      C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.wrn
      C:\Users\gotos\source\repos\runtime\src\coreclr\..\..\artifacts\log\Debug\Cross_Linux__x64__Debug.err
C:\Users\gotos\source\repos\runtime\src\coreclr\runtime.proj(32,5): error MSB3073: ""C:\Users\gotos\source\repos\runtime\src\coreclr\build-runtime.cmd" -x64 -debug -linuxdac" 명령이 종료되었습니다(코드: 1).

빌드하지 못했습니다.

C:\Users\gotos\source\repos\runtime\src\coreclr\runtime.proj(32,5): error MSB3073: ""C:\Users\gotos\source\repos\runtime\src\coreclr\build-runtime.cmd" -x64 -debug -linuxdac" 명령이 종료되었습니다(코드: 1).
    경고 0개
    오류 1개

경과 시간: 00:01:13.40
Build failed.
Some builds failed:
        Configuration: Debug, Architecture: x64
```
</details>

--- a/src/coreclr/src/pal/src/libunwind/config.h.in
+++ b/src/coreclr/src/pal/src/libunwind/config.h.in
@@ -17,4 +17,13 @@
 
 #cmakedefine HAVE_ATOMIC_OPS_H
 
+#cmakedefine HAVE_STDALIGN_H
+#cmakedefine HAVE_STDALIGN_ALIGNAS
+
+#if defined(_MSC_VER) && defined(HAVE_STDALIGN_H) && !defined(HAVE_STDALIGN_ALIGNAS)
+// alignment is a performance optimization for the cross compile libunwind
+// Simply ignore it if it is not supported by the compiler
+#define alignas(x)
+#endif
+
 #endif

--- a/src/coreclr/src/pal/src/libunwind/configure.cmake
+++ b/src/coreclr/src/pal/src/libunwind/configure.cmake
@@ -59,6 +59,16 @@ int main(int argc, char **argv)
     return 0;
 }" HAVE__BUILTIN_UNREACHABLE)
 
+check_c_source_compiles("
+#include <stdalign.h>
+
+int main(void)
+{
+    alignas(128) char result = 0;
+
+    return result;
+}" HAVE_STDALIGN_ALIGNAS)
+
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/config.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/config.h)
 add_definitions(-DHAVE_CONFIG_H=1)
 

